src/commands/diff.ts

```diff
@@ -125,8 +128,11 @@ export default class Diff extends Command {
         err: error,
       });
     }
+    if (watchMode) {
+      specWatcher({spec: firstDocument, handler: this, handlerName: 'diff', docVersion: 'old'});
+      specWatcher({spec: secondDocument, handler: this, handlerName: 'diff', docVersion: 'new'});
```

Comment:
Here we have a serious error. As we create a watcher only once, a second watcher will not be created at all. We need to be able to create multiple watchers. We suggest to add a label to the watcher and then create/check if it exists (we can use map to store the watchers):

```ts
if (watcherMap.has({Label})) {
  return
}

...
watcherMap.set({Label}, {Watcher})
```

Reply:
Yeah, I noticed this. Just built a nice store for it :)